gravitee-am-ui/package.json

```diff
@@ -77,23 +77,33 @@
     "zone.js": "0.11.8"
   },
   "devDependencies": {
-    "@angular-builders/jest": "11.2.0",
-    "@angular-devkit/build-angular": "13.3.11",
-    "@angular/cli": "13.3.11",
-    "@angular/compiler-cli": "13.4.0",
-    "@angular/language-service": "13.4.0",
-    "@types/jest": "25.2.3",
+    "@angular-builders/jest": "14.1.0",
+    "@angular-devkit/build-angular": "14.2.13",
+    "@angular-eslint/builder": "14.4.0",
+    "@angular-eslint/eslint-plugin": "14.4.0",
+    "@angular-eslint/eslint-plugin-template": "14.4.0",
+    "@angular-eslint/schematics": "14.4.0",
+    "@angular-eslint/template-parser": "14.4.0",
+    "@angular/cli": "14.2.13",
+    "@angular/compiler-cli": "14.3.0",
+    "@angular/language-service": "14.3.0",
+    "@storybook/addon-essentials": "6.4.17",
+    "@storybook/angular": "6.4.17",
+    "@types/jest": "28.1.8",
     "@types/lodash": "4.14.196",
     "@types/node": "12.20.55",
```

Comment:
should it be node 16?

Reply:
yes. 
In fact i have a 2nd PR in preparation (for angular 15) which updates all the points below and much more. it will also pass from dependence to dev dependence and remove those which are useless.
And I finally cut it in 2 to have a 1st step.